Compiler.Lexer/Compiler.Lexer/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Compiler.Lexer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Tokenizer tokenizer = new Tokenizer(args[0]);
```

Comment:
No Exception handlers in code, debug a issue may result in a full stacktrace search